app/components/containerInspect/containerInspect.html

```diff
@@ -0,0 +1,22 @@
+<rd-header>
+  <rd-header-title title="Container inspect">
+    <i id="loadingViewSpinner" class="fa fa-cog fa-spin" ng-if="state.loaded === false"></i>
+  </rd-header-title>
+  <rd-header-content ng-if="state.loaded">
```

Comment:
No need to check for `state.loaded` here. Have a look at my comment regarding `initView()` below.